Eurovision Song Contest Analysis/Model/README.md

```diff
@@ -0,0 +1,45 @@
+
+
+**Eurovision Song Contest Anlaysis**
+
+**GOAL**
+
+To analyze the Eurovision Dataset using Exploratory Data analysis.
+
+**DATASET**
+
+https://www.kaggle.com/datasets/latifahmakuyi/eurovision-winners-1956-2022
+
+**DESCRIPTION**
+
+Eurovision Song Contest is an international song competition organised annually.The goal of this project to analyze the data between 1956 and 2022 and derive meaningful conclusions.
+
+**WHAT I HAD DONE**
+
+* Checked for missing values and cleaned the data accordingly
+* Analyzed the data, found insights and visualized them accordingly.
+* Found detailed insights of different columns with one another using plotting libraries.
+
+
+**LIBRARIES NEEDED**
+
+1. Pandas
+2. Matplotlib
+3. Seaborn
+4. NumPy
+
+**VISUALIZATION**

```

Comment:
Add your EDA/Visualization outputs under this **Visualization** heading.

Reply:
added the images.